crypto/bn/bn_dh.c

```diff
@@ -45,9 +45,23 @@ static const BN_ULONG modp_1536_p[] = {
     BN_DEF(0x8A67CC74, 0x29024E08), BN_DEF(0x80DC1CD1, 0xC4C6628B),
     BN_DEF(0x2168C234, 0xC90FDAA2), BN_DEF(0xFFFFFFFF, 0xFFFFFFFF)
 };
+/* q = (p - 1) / 2 */
```

Comment:
Blank lines between these blocks of numbers perhaps?

Reply:
This is a deliberate grouping.

Reply:
The reasonable.  However `dh2048_224_g` has a (pre-existing) blank line before it :(  I didn't notice any others.